### DISCUSSION TITLE
Do not add the game bgm inside the as project

### DISCUSSION THREAD

**Discussion by ttttttttttttttllllllllllllll** [Tualin14/baPlayer#25] [General]
@kiraio-moe 
![image](https://github.com/Tualin14/ArisStudio/assets/71857734/33525938-1962-4127-b5ea-aec79c8cf134)

**Comment by kiraio-moe** [comment]
So we can't make main menus in Aris Studio like MomoTalk?
So what should we do?

Of course changing the MomoTalk logo with the gear logo as before is one  of them.

**Reply by ttttttttttttttllllllllllllll** [reply]
I think it's because officials don't want third-party-generated content to be considered the game's own when it's too similar to the game.

So we may have to go and slightly modify the ui, performance effects, etc.

**Reply by ttttttttttttttllllllllllllll** [reply]
Let's ignore this issue for now, no one mentioned this and left it alone.

**Reply by kiraio-moe** [reply]
Changing the UI is fine, but changing other aspect like emoticon animation is way more far away from this repository goal.

**Reply by ttttttttttttttllllllllllllll** [reply]
Yes, so the current plan is to leave the matter alone. 
After all, we have a final solution to let users build their own projects.

**Reply by kiraio-moe** [reply]
Where did you get the notice from?

**Reply by ttttttttttttttllllllllllllll** [reply]
This is the official warning received by the author of Mollutalk (a momotalk maker).

Also mention that I deleted git history before because [blue-archive-data-download](https://github.com/nijinekoyo/blue-archive-data-download) was officially warned, so our project should use as little material from the game as possible in non-essential cases. (Let users get it on their own).

Let's not focus on that right now, but be prepared to deal with it.

**Comment by ttttttttttttttllllllllllllll** [comment]
Ok. I know the reason Mollutalk was banned was the inclusion of the ability to use the game's bgm, so don't go using those things inside arisstudio.

**Reply by kiraio-moe** [reply]
Ok, I will remember that.
